packages/forms/lib/src/timepicker/README.md

```diff
@@ -16,8 +16,11 @@ Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to
 
 | Name         | Default value | Description |
 | -----------  | ------ | -------------------------- |
-| `@Input() hoursPlaceholder: string;` | `'HH'` | Optional placeholder used in the hour picker. |
+| `@Input() hoursPlaceholder: string;` | `'UU'` | Optional placeholder used in the hour picker. |
```

Comment:
zijn mensen zelfs in het nederlands niet meer HH gewend?

Reply:
Guideline is dat alles Nederlands moet zijn. Ik denk dat UU:MM evenveel gebruikt wordt.